sdk/storage/azure-storage-file/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## Version 12.0.0 (2019-10-31)
```

Comment:
File will not go with GA but another round of preview. Might not need to change here.